src/core/lib/iomgr/socket_utils_common_posix.cc

```diff
@@ -222,6 +222,32 @@ grpc_error* grpc_set_socket_low_latency(int fd, int low_latency) {
   return GRPC_ERROR_NONE;
 }
 
+#define DEFAULT_TCP_USER_TIMEOUT 20000 /* 20 seconds */
```

Comment:
Add the `MILLIS` suffix to the name  i.e `DEFAULT_TCP_USER_TIMEOUT_MILLIS`

Reply:
changed the suffix to _MS following code at other places